actors/builtin/miner/miner_actor.go

```diff
@@ -38,7 +38,7 @@ const (
 
 type CronEventPayload struct {
 	EventType CronEventType
-	Sectors   *abi.BitField
+	Sectors   bitfield.BitField
```

Comment:
Should this one still be by-pointer? Technically, an uninitialized bitfield will correctly serialize to an empty bitfield, so we don't _have_ to make this nullable.

Reply:
This is fine. I think we'll be able to delete this soon anywa.